docs/index.rst

```diff
@@ -26,16 +26,8 @@ Our documentation is structured as follows:
   * :ref:`install-linux-quick`
   * :ref:`install-linux`
 
-Contributing to this documentation
+Related information
 =======================================================
 
-We welcome collaboration! If you'd like to contribute to our documentation, you can find instructions
-in our `Contributing to ROCm  <https://rocm.docs.amd.com/en/latest/contribute/index.md>`_ section, which discusses:
-
-* `Document structure <https://rocm.docs.amd.com/en/develop/contribute/index.html#supported-formats>`_
-* `Toolchains <https://rocm.docs.amd.com/en/develop/contribute/toolchain.html>`_
-* `Documentation builds <https://rocm.docs.amd.com/en/develop/contribute/building.html>`_
-* `How to provide feedback <https://rocm.docs.amd.com/en/develop/contribute/feedback.html>`_
-
-Licensing information for all ROCm components is listed on our
```

Comment:
If you don't like licensing info lumped in with the contribution stuff, we can add a new heading.

Reply:
Hi Lisa, I updated this as we discussed. I removed the header, and removed the bullets, and restructured the sentences.